pkg/trace/api/profiles.go

```diff
@@ -34,6 +34,13 @@ func profilingEndpoints(apiKey string) (urls []*url.URL, apiKeys []string, err e
 	main := profilingURLDefault
 	if v := config.Datadog.GetString("apm_config.profiling_dd_url"); v != "" {
 		main = v
+		if strings.HasSuffix(main, "/v1/input") {
```

Comment:
These strings should probably be constants if we're going to use them in multiple places.